subprojects/logging/src/main/java/org/gradle/internal/logging/progress/ProgressLoggerFactory.java

```diff
@@ -36,5 +38,16 @@
      */
     ProgressLogger newOperation(Class<?> loggerCategory);
 
+    /**
+     * Creates a new long-running operation which has not been started, associated
+     * with the given build operation id.
+     *
+     * @param loggerCategory The logger category.
+     * @param buildOperationId OperationIdentifier to be associated with all logs produced by
```

Comment:
I am not sure if the Javadoc would render properly if the parameter description is on two lines.